### PR TITLE
Adds engineering optional objectives

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR adds an objective to never have an engine meltdown, either SD or RBMK, and also adds an objective to produce randomly between 60-90% of 13 MW for the SD and 10 MW for the RBMK

## Why It's Good For The Game

Engineers really need more objectives

## Changelog
:cl:
add: Added new objectives for engineer, one to have a certain power output by roundend, and the other to never have an engine meltdown (SD or RBMK)
/:cl:

### CODE REVIEW

**Comment by MacBlaze1** [comment]
> Are these numbers rounded to a certain position?

elaborate please

File: nsv13/code/modules/crew_objectives/engineering_objectives.dm [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*				ENGINEERING OBJECTIVES				*/
+
+/datum/objective/crew/integrity //ported from old Hippie
```
Comment (Karmic-Skink) [inline nsv13/code/modules/crew_objectives/engineering_objectives.dm:3]: Why not include something about our overmap state here?

Comment (MacBlaze1) [reply nsv13/code/modules/crew_objectives/engineering_objectives.dm:3]: uhhh,  what do you mean

Comment (Karmic-Skink) [reply nsv13/code/modules/crew_objectives/engineering_objectives.dm:3]: Hull Integrity, Armour quadrants etc.

Comment (MacBlaze1) [reply nsv13/code/modules/crew_objectives/engineering_objectives.dm:3]: wait you're saying we dont already have that objective?

**Review by Kmc2000 — APPROVED** [review]

**Pull request merged**